src/test/scala/com/bwsw/cloudstack/vault/server/kafka/ConsumerTestSuite.scala

```diff
@@ -108,4 +108,31 @@ class ConsumerTestSuite extends FlatSpec with Matchers {
     assert(consumer.process().isInstanceOf[Unit])
   }
 
+  "process" should "does not swallowed non-CriticalException" in {
```

Comment:
"process" should "not swallow non-CriticalException"

Reply:
ok